content/inside-rust/clippy-warming-up/index.md

```diff
@@ -0,0 +1,60 @@
++++
+path = "inside-rust/9999/12/31/clippys-feature-warming-up"
+title = "What we learned with Clippy's feature freeze"
+authors = ["blyxyas"]
+
+[extra]
+team = "the Clippy team"
+team_url = "https://www.rust-lang.org/governance/teams/dev-tools#team-clippy"
++++
+
+
+The [feature freeze affecting Clippy][feature-freeze] has reached its end date. This means that the Clippy project has been accepting pull
+requests that add new features (and new lints) for a while.
+
+The Clippy team has reviewed the results, and we can say that it's been a success 🎉. It has been so great that we managed
+to create an all-time peak in the number of pull requests opened in a week.
+
+In this plot we can observe that peak (the red lines mark the start and end of the feature freeze)
+
+![Plot of PRs per week, we can see an all-time peak when the feature freeze happens](/inside-rust/clippy-warming-up/prs_per_week.png)
+
+After some data crunching of those pull requests opened after June 26th, we had 18 pull requests open in that period
+that added lints. With 326 pull requests open by both and old contributors, we want to highlight all the new people
```

Comment:
```suggestion
that added lints. With 326 pull requests open by both new and old contributors, we want to highlight all the new people
```
Missing a word like this?

Reply:
Fixed on https://github.com/rust-lang/blog.rust-lang.org/pull/1726/commits/028a83d1e43d54cf5e3b644e397a1af9b711146d